code/game/machinery/bots/farmbot.dm

```diff
@@ -187,7 +187,7 @@
 	src.icon_state = "[src.icon_initial][src.on]"
 	target = null
 	mode = FARMBOT_MODE_WAITING //Give the emagger a chance to get away! 15 seconds should be good.
-	spawn(150)
+	spawn(15 SECONDS)
 		mode = 0
```

Comment:
no longer necessary because of the frustration rework, also several srcs to remove above this

Reply:
can it even attack or do anything evil when emagged if it can't move to its target

Reply:
Yeah it avoids moving at all and just throws fertilizer bottles

Reply:
those bottles should be ever so slightly explosive
ammonium nitrate and what not